frontend/src/Components/Page/Header/ArtistSearchInput.css

```diff
@@ -72,7 +72,7 @@
 }
 
 .highlighted {
-  background-color: $themeLightColor;
+  background-color: rgba(0, 0, 0, 0.4);
```

Comment:
Don’t hard code here, issue is that I have themeDarkColor and themeLightColor set to same color. Let’s correct it there. Frontend/src/styles/variables/colors.js

Reply:
I tried both dark and light color neither of them were working